### ISSUE TITLE
Validation error

### ISSUE THREAD

**Issue by Yippee38** [wabbajack-tools/wabbajack#1220]
**Describe the bug**

My existing game got messed up due to HDD problems.  I am trying to re-install the build I had, using the same version of WJ and the original .wabbajack file.  I created a new directory (backup up original) and copied the "downloads" directory from the original install.  That directory hasn't been touched since the original install.  My thought was to run Wabbajack again to re-install the build that I had before.  Since all of the files are already downloaded, I expected WJ to see that, not download anything, but run the install process.

When I attempted this, I got a "validation error" that prevented me from continuing.  The .log file doesn't help me much.  I do see a line that says one of the download files is not whitelisted.  My build was a Lexy's LOTD build, which is no longer supported.  Is that why the whitelisting is failing?  Is the whitelist local (part of the .wabbajack file), or hosted somewhere else?

Is there any way I can re-install this build at this point?  If so, what do I need to do?

**Additional Context**

Windows 10 Home
Skyrim SE

**Wabbajack Version**

I'm using Wabbajack v0.9.16.0
I know this is an old version.  Like I said above, this is the version I used for my initial install, so I didn't want to change it.

**Logs**

0.07 - Wabbajack Build - 129fb445b8b5b05c6e6200c97aa0a44211f02071
8.3 - Benchmarking E:\Game Utilities\Skyrim SE\Wabbajack
10.44 - 966.4MB/sec for E:\Game Utilities\Skyrim SE\Wabbajack
10.44 - Recommending a queue size of 8 based on disk performance, number of cores, and 31.8GB of system RAM
10.44 - Cleaning VFS, this may take a bit of time
10.49 - Steam Library found at F:\Steam\steamapps
10.49 - Total number of Steam Libraries found: 1
10.49 - Searching for Steam Games in F:\Steam\steamapps
##a bunch of Steam games##
10.6 - Found Steam Game: "The Elder Scrolls V: Skyrim" (72850) at e:/steam\steamapps\common\Skyrim
##yet more Steam games##
10.6 - Total number of Steam Games found: 4
10.61 - Found 39 SubKeys for GOG
10.61 - GOG Game "Fallout" (1) is not supported, skipping
##a bunch more GOG games##
10.64 - Total number of GOG Games found: 0
10.64 - Validating Game ESMs
10.65 - Validating Dawnguard.esm
10.76 - Validating Dragonborn.esm
10.99 - Validating HearthFires.esm
11 - Validating Update.esm
11.07 - Validating Modlist
11.07 - Loading Nexus mod permissions
11.23 - Loaded permissions for 1 authors
11.23 - Loading server whitelist
11.26 - Loaded permissions for 271 servers and 99 Google Drive files
11.26 - Running validation checks
11.55 - Finishing Line Files.7z is not a whitelisted download
11.57 - Encountered error, can't continue - System.Exception: 1 validation errors found, cannot continue.
   at Wabbajack.Lib.Validation.ValidateModlist.<RunValidation>d__13.MoveNext() in d:\work\1\s\Wabbajack.Lib\Validation\ValidateModlist.cs:line 72
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.GetResult()
   at Wabbajack.Lib.MO2Installer.<_Begin>d__11.MoveNext() in d:\work\1\s\Wabbajack.Lib\MO2Installer.cs:line 71
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Wabbajack.Lib.ABatchProcessor.<<Begin>b__49_0>d.MoveNext() in d:\work\1\s\Wabbajack.Lib\ABatchProcessor.cs:line 172
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Wabbajack.MO2InstallerVM.<>c__DisplayClass31_0.<<Install>b__0>d.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Wabbajack.MO2InstallerVM.<Install>d__31.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at Wabbajack.InstallerVM.<>c__DisplayClass101_0.<<-ctor>b__70>d.MoveNext()

**Comment by EzioTheDeadPoet** [comment]
I recommend trying the latest WJ version as well.

**Comment by Yippee38** [comment]
I can certainly do that.  Do you know about the whitelist?  Where is the whitelist?  Can I alter it?

**Comment by tr4wzified** [comment]
You cannot install Lexy's LOTD anymore, it's been taken down for quite some time now.

**Comment by EzioTheDeadPoet** [comment]
well he said he has all the downloads and the original .wabbajack 🤔

**Comment by LostDragonist** [comment]
The whitelist is hosted here: https://github.com/wabbajack-tools/opt-out-lists

It's not really going to be possible to change it yourself to make the install work.  It may be possible for WJ to be modified to only check the whitelist when installing files that are already there though bossman would have to make the call on if that negatively affects the intent of the whitelist.  (Someone would also have to make the change which may or may not happen).

**Comment by Yippee38** [comment]
> I recommend trying the latest WJ version as well.

The new version is saying that the modlist is corrupt.

> The whitelist is hosted here: https://github.com/wabbajack-tools/opt-out-lists
> 
> It's not really going to be possible to change it yourself to make the install work. It may be possible for WJ to be modified to only check the whitelist when downloading files though bossman would have to make the call on if that negatively affects the intent of the whitelist. (Someone would also have to make the change which may or may not happen).

I get the concept of that, but it seems weird to me that I can't re-install an old modlistf given how long a playthrough on any of the games supported can take.  It would be cool to have the option to use a locally cached whitelist, or just a local copy.  Obviously, that wouldn't be supported.  But my only option now is to throw away the hours I've spent on my latest playthrough and start over with a new modpack, or manually install Lexy's LOTD (which would easily take me more than a week).  Either way, my playthrough is lost because I can't re-install the modpack that I had already installed.  I know it's a one-off kind of situation, but it seems strange that that's not an option.
